R/nneo_wrangle.R

```diff
@@ -0,0 +1,108 @@
+#############################################################################################
+#' @title Download and wrangle NEON's sensor-based data products
+
+#' @author Josh Roberti \email{jaroberti87@gmail.com}\cr
```

Comment:
Sorry, just referring to the email address, not the roxygen tags (e.g.,  `@param` or `@author`)